### PR TITLE
fix: adjustments to view component family and more

### LINKED ISSUE #4765: 'view chart' not working
https://biosimulations.org/runs/64ef9aee28f96276a163d941

'Select chart':

image

After clicking 'view':

image

### LINKED ISSUE #4767: 'simulation specification' tab probably lying about having no image to display
https://run.biosimulations.org/runs/64e5319b548ceb84f3ad843c

On the 'simulation specification' tab, it claims there's no image to display:

image

However, I'm almost certain there are indeed images to display there.

### LINKED ISSUE #4768: Clicking on tabs should give new URL
It is useful to be able to provide links like

https://run.biosimulations.org/runs/64e5319b548ceb84f3ad843c#tab=log

so you can go directly to that page.   This happened on the old production site, but not the new one.  Now, you just get the base URL regardless of what tab you're on:

https://run.biosimulations.org/runs/64e5319b548ceb84f3ad843c

### LINKED ISSUE #4771: Auto-expand all 'details' information for projects
Right now, two of the five categories are auto-expanded when you click on 'details':

image

However, none of the exposed information is particularly interesting (IMO).  I would put 'Identifiers, citations, and license' at the top, and expand it, then the 'Methods and Tools', expanded, then the Model/simulation, expanded.  

Finally, I'd put  'simulation run' and 'provenance' at the bottom, and probably go ahead and expand them?  But if you like it a bit more compact, then those are the ones I would leave collapsed.

image

### LINKED ISSUE #4774: Graphs cut off legend
From https://biosimulations.org/projects/BIOMD0000000010_tellurium_Negative_feedback_and_ultrasen

image

The legend in the upper right is cut off.

### PR DESCRIPTION
**Pull request by AlexPatrie**
**What new features does this PR implement?**
Please summarize the features that this PR implements. As relevant, please indicate the issues that the PR closes.

* `platform`/`dispatch`: implements updated two-way event and property binding for `.runs.view` within `mat-tab-group`. (Fixes #4768)
* `platform`/`dispatch`: implements new logic for controlling which `mat-expansion-panels` render pre-opened in `.runs.view` for simulation project details. (Fixes #4771).

**What bugs does this PR fix?**
Please summarize the bugs that this PR fixes. As relevant, please indicate the issues that the PR closes.

* `platform`/`dispatch`: adjusts `mat-card` widths properly in `sideBar` selector for `.runs/view`.
* `platform`: adjusts `.plotly-visualization` layout declaration in `./projects/view` to properly scale with entire content width. (Fixes #4774)

**Does this PR introduce any additional changes?**
Please summarize any additional changes that this PR introduces.

* Adds deeper `...-darker-color` uniquely for both `platform`/`dispatch` and `simulators` apps on `:hover` in `biosimulations-navigation` hero banner nav buttons.

**How have you tested this PR?**
Please summarize the tests that you implemented to test this PR.

* CI/CD pipeline testing.
* SonarCloud CI.
* CodeFactor CI.

**Additional information**
This serves as the second UI adjustment following a deployment after nearly a year of ui-dev stagnation.

### CODE REVIEW

File: apps/simulators/src/styles.scss [hunk 1]
```diff
@@ -92,3 +92,10 @@
   display: flex;
   align-items: center;
 }
+
+biosimulations-hover-open-menu button:hover {
+  background-color: var(--primary-darker-color) !important;
+  transition: background-color 0.35s ease !important;
+  color: white !important;
+  font-weight: 700;
+}
```
Comment (eagmon) [inline apps/simulators/src/styles.scss:101]: why add it here if pretty much the same thing is going in `_global.scss`?

**Review by eagmon — APPROVED** [review]
Looks mostly good to me. Just one question below about repeated code in `apps/simulators/src/styles.scss` and `libs/shared/styles/src/lib/_global.scss`. But if you think that's correct, you can go ahead and merge.

File: apps/simulators/src/styles.scss (hunk 1 above)
Comment (AlexPatrie) [reply apps/simulators/src/styles.scss:101]: There are several scopes to consider when discussing repo styling: global, app-wide, local (to component), and In-line. This serves as an app-wide override for just `simulators` which will pierce the shadow-DOM only for this app...it is the property controller of navigation buttons `background-color`.

**Pull request merged**